aries_cloudcontroller/controllers/issuer.py

```diff
@@ -135,8 +135,10 @@ async def issue_credential(self, cred_ex_id, comment, attributes):
         )
 
     # Store a received credential
-    async def store_credential(self, cred_ex_id, credential_id):
-        body = {"credential_id": credential_id}
+    async def store_credential(self, cred_ex_id, credential_id: str = None):
```

Comment:
This should not be in this commit. It is also added in separate P.R. #21

Reply:
I'll address that

Reply:
@wip-abramson just looked at that and now I'm confused what you mean. The PR you referenced is this PR? Not sure what you mean now. Issue is 14 PR is 21. Happy to change however you like it, please, clarify though😅